### ISSUE TITLE
Issue: ModuleNotFoundError: No module named 'six'

### ISSUE THREAD

**Issue by imaxmelnyk** [langchain-ai/langsmith-sdk#1195]
### Issue you'd like to raise.

```
from langchain_core.language_models.chat_models import LangSmithParams
venv/lib/python3.9/site-packages/langchain_core/language_models/__init__.py:50: in <module>
    from langchain_core.language_models.chat_models import BaseChatModel, SimpleChatModel
venv/lib/python3.9/site-packages/langchain_core/language_models/chat_models.py:33: in <module>
    from langchain_core.callbacks import (
venv/lib/python3.9/site-packages/langchain_core/callbacks/__init__.py:23: in <module>
    from langchain_core.callbacks.manager import (
venv/lib/python3.9/site-packages/langchain_core/callbacks/manager.py:23: in <module>
    from langsmith.run_helpers import get_tracing_context
venv/lib/python3.9/site-packages/langsmith/run_helpers.py:44: in <module>
    from langsmith import client as ls_client
venv/lib/python3.9/site-packages/langsmith/client.py:69: in <module>
    from langsmith import env as ls_env
venv/lib/python3.9/site-packages/langsmith/env/__init__.py:3: in <module>
    from langsmith.env._runtime_env import (
venv/lib/python3.9/site-packages/langsmith/env/_runtime_env.py:10: in <module>
    from langsmith.utils import get_docker_compose_command
venv/lib/python3.9/site-packages/langsmith/utils.py:43: in <module>
    from langsmith._internal import _patch as patch_urllib3
venv/lib/python3.9/site-packages/langsmith/_internal/_patch.py:4: in <module>
    import six  # noqa
E   ModuleNotFoundError: No module named 'six'
```

version: 0.1.141
was ok on 0.1.140

**Comment by nachonavarro** [comment]
This is urgent, it breaks langchain as well (e.g. when using a `JsonOutputParser`).

**Comment by efriis** [comment]
fixed in #1197

**Comment by hinthornw** [comment]
Yanked release
